caddyconfig/httpcaddyfile/options.go

```diff
@@ -174,6 +174,18 @@ func parseOptACME(d *caddyfile.Dispenser) (string, error) {
 	return val, nil
 }
 
+func parseOptACMECARoot(d *caddyfile.Dispenser) (string, error) {
+	d.Next() // consume parameter name
+	if !d.Next() {
+		return "", d.ArgErr()
+	}
+	val := d.Val()
+	if d.Next() {
+		return "", d.ArgErr()
+	}
+	return val, nil
+}
+
```

Comment:
Since this is the exact same as `parseOptACME`, just use `parseOptACME` instead.